woocommerce/woocommerce-android
# Take all screenshots

Adds the code to take all the screenshots on the Espresso side as well as a Fastlane lane to take them.

### To test

Run

```
bundle install install --with-screenshots \
  && bundle exec fastlane screenshots \
  locales:en,it # this is so they don't take forever, feel free to use any locale
```

and verify that the screenshots come out as expected in `fastlane/screenshots/en/images/phoneScreenshots`

![abc-screenshot-2020-06-03-212247](https://user-images.githubusercontent.com/1218433/83631160-69880280-a5e0-11ea-9aa4-139f25dbeb44.png)

---

Update release notes:
